src/create-node-middleware.ts

```diff
@@ -10,7 +10,7 @@ export function createNodeMiddleware(
 ): RequestListener {
   probot.load(appFn);
 
-  return createWebbhooksMiddleware(probot.webhooks, {
+  return createWebhooksMiddleware(probot.webhooks, {
     path: webhooksPath || "/",
```

Comment:
This should probably be updated to use the same path as in `@octokit/webhooks`
```suggestion
    path: webhooksPath || "/api/github/webhooks",
```

Reply:
done

Reply:
done